components/form/Select.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ErrorMessage } from './ErrorMessage';
+import { useFormContext } from 'react-hook-form';
+
+export function SelectField({ label, name, ...options }) {
```

Comment:
We would also want to add a `className` here to customize the Select component. I think for all of the form components, we would want to add the `className`, `containerClassName` and `labelClassName` to provide endless customization.